target/linux/ipq806x/files-4.14/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -319,25 +319,14 @@
 
 					ubi@1880000 {
 						label = "ubi";
-						reg = <0x1880000 0x1C00000>;
-					};
-
-					netgear@3480000 {
-						label = "netgear";
-						reg = <0x3480000 0x4480000>;
-						read-only;
+						reg = <0x1880000 0x6080000>;
```

Comment:
Are you sure it really works? I'm quite sure ubi either goes crazy or refuses to use the additional space if it isn't properly ubi formatted.

I mean, we have the EOF marker (to trigger a ubiformat) for ubinized images for a reason.

Reply:
I have been using that in master for a month now, the same change is already in Openwrt master sources.

And I have been using it in my own 18.06 community build for R7800 already for the same month.

If you look at the links in my original PR #1003 , that "Netgear" area is an ubi formatted partition in the Netgear OEM firmware (and there it gets split into six smaller partitions).

Reply:
> If you look at the links in my original PR #1003 , that "Netgear" area is an ubi formatted 

That explains why it works. Thanks for the explanation.

Reply:
I am no ubi specialist, but so far it has just worked ;-)

Btw, the hack has been used by some users ever since early 2017 (when I figured out the needed change):
https://forum.lede-project.org/t/netgear-r7800-only-19mb-flash-available/1115